Doc/library/unittest.mock-examples.rst

```diff
@@ -512,16 +521,6 @@ call::
     >>> something.backend = mock_backend
     >>> something.method()
 
-Using :attr:`~Mock.mock_calls` we can check the chained call with a single
```

Comment:
Why do you remove this? This works fine.

Reply:
...because it's not safe to use: because the ancestor parameters are not compared, you can easily be fooled into thinking something is being called correctly when it is not.

Reply:
Maybe I am misunderstanding this whole thing, but what is not?

just try changing foobar here to `foobaz`, the assert will fail because this is using `call_list`

Reply:
I don't mind a note being added, although perhaps just the note without the warning. ``call_list`` is still useful so I don't want to see this removed from the docs.